packages/bsky/src/db/tables/did-cache.ts

```diff
@@ -0,0 +1,13 @@
+import { DidDocument } from '@atproto/did-resolver'
+
+export interface DidCache {
+  did: string
+  doc: DidDocument
```

Comment:
Is it all good for tombstoned dids to be cached here?

Reply:
I cleaned it up a bit so if we try to refresh & it's missing, we'll clear out the cache. should fix tombstoned dids 👍